src/main/java/com/jungdam/member/domain/vo/Avatar.java

```diff
@@ -14,10 +19,17 @@ protected Avatar() {
     }
 
     public Avatar(String avatar) {
-        this.avatar = avatar;
+        this.avatar = validate(avatar);
+    }
+
+    private String validate(String avatar) {
+        if (Objects.isNull(avatar)) {
+            return NO_AVATAR;
+        }
+        return avatar;
     }
 
-    public String getAvatar() {
+    public String validate() {
```

Comment:
```suggestion
    public String getAvatar() {
```
메소드 명이 이렇게 되는게 맞는 것 아닌가요..?

Reply:
엇 저도 `getAvatar()`가 맞을 것 같아요

Reply:
앗 이것은 극락의 실수~~
수정하겠슴돠~